src/ZstdSharp/ZstdSharp.csproj

```diff
@@ -35,7 +35,7 @@
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="5.0.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.1'">
+  <ItemGroup Condition="'$(TargetFramework)'!='net7.0'">
```

Comment:
Should be all frameworks below `.net5.0`, now it's `net6.0` and `net8.0` and future frameworks that are not needed.

Reply:
Done.